handbook/ops/bizops/onboarding.md

```diff
@@ -0,0 +1,61 @@
+# BizOps onboarding
+
+Welcome! We're so excited to have you join the BizOps team. This document outlines the structure of your first few weeks at Sourcegraph.
+
+## Week 1
+
+### Get set up
+
+You'll have to get some basics set up in your first few days:
+
+- Complete the [general onboarding](../../people-ops/onboarding/general_onboarding.md). 
+
+### Understand company and team mission, values and goals
+
+- [Sourcegraph FY22 OKRs](https://docs.google.com/document/d/18d3sX38O6ephNuoHqZT9BwU1m1_FGuRfsTwYbc8lMV8/edit#heading=h.obnaanj0svtb)
+- [Sourcegraph values](../../../company/values.md)
+- [Sourcegraph master plan](../../../company/strategy.md)
+- [Sourcegraph direction (1 year plan)](../../../direction/index.md)
+- [Current BizOps projects](https://github.com/orgs/sourcegraph/projects/63)
+- [BizOps @ Sourcegraph deck](https://docs.google.com/presentation/d/1B8TIkOsFXUofLKcwu3ZKUuFgSe7si_HNFXj8Qne4_ug/edit#slide=id.gb969c13690_0_15)
+
+### Understand Sourcegraph communication and documentation standards
```

Comment:
This section is great. Did you write it from scratch, or pull it from another onboarding? We should share it if the former.

Reply:
From scratch but probably with some inspiration from other team-specific onboardings. Honestly can't remember :)